src/track/trackplaytimers.cpp

```diff
@@ -0,0 +1,50 @@
+#include "track/trackplaytimers.h"
+
+TrackTimers::GUITickTimer::GUITickTimer() 
+        :  m_msSoFar(0.0),
+           m_msTarget(0.0),
+           m_isActive(false),
+           m_timeoutSent(false) {
+    
+}
+
+void TrackTimers::GUITickTimer::start(int msec) {
+    m_msTarget = static_cast<double>(msec);
```

Comment:
Why not allow to start the timer with a double?

Reply:
Done.